<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmsInfo</class>
 <widget class="QWidget" name="EmsInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Interface Version:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="interfaceVersionLineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>141</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Firmware Version:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="firmwareVersionLineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="locationIdInfoTableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>781</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
